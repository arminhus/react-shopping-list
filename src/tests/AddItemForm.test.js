import React from "react";
import Enzyme from "enzyme";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

import AddItemForm from "../components/AddItemForm";

Enzyme.configure({ adapter: new Adapter() });

describe("AddItemFrom Component Test", () => {
  test("Should render the complete form", () => {
    const wrapper = shallow(<AddItemForm />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
