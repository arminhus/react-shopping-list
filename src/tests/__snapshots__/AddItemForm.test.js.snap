// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddItemFrom Component Test Should render the complete form 1`] = `
<form>
  <input
    aria-label="Shopping list item"
    placeholder="carrots"
    type="text"
  />
  <button
    type="submit"
  >
    Add Item
  </button>
</form>
`;
